glean_parser/metrics.py

```diff
@@ -306,6 +306,13 @@ class Labeled(Metric):
     labels: List[str] = None
     labeled = True
 
+    def __post_init__(self, *args):
+        ordered_labels = self.labels
```

Comment:
Instead of doing this, one other option could be to check if the metric type is a LabeledMetricType and do the fix-up [here](https://github.com/mozilla/glean_parser/blob/83d3af92354569cb9c87a1b1851f342397c4a234/glean_parser/kotlin.py#L19). @mdboom , do you have opinions on any of the two solutions?

Reply:
I think this is cleaner, and it will apply to all output formats.

Reply:
Then @brizental let's add a comment here explaining what is this for, please :)

Reply:
In this context, does `this` mean my fix os Alessio's suggestion?

Reply:
Sorry.  I think @brizental's fix using the `__post_init__` method is cleaner.